srcpkgs/refind/template

```diff
@@ -73,10 +73,7 @@ do_install() {
 	vinstall gptsync/gptsync_${_EFI_ARCH}.efi 644 \
 		usr/share/refind/tools_${_EFI_ARCH}/
 
-
 	vinstall "${FILESDIR}/refind-kernel-hook.conf" 644 etc/default
-	vinstall ${FILESDIR}/kernel.post-install 744 \
-		etc/kernel.d/post-install 50-refind
-	vinstall ${FILESDIR}/kernel.post-remove 744 \
-		etc/kernel.d/post-remove 50-refind
+	vinstall "${FILESDIR}/kernel.post-install" 744 etc/kernel.d/post-install 50-refind
+	vinstall "${FILESDIR}/kernel.post-remove" 744 etc/kernel.d/post-remove 50-refind
```

Comment:
These changes should be dropped. The line continuations are a bit easier to read and honor the preferred 80-column text limit.

Reply:
I like the REFIND_MENU_LABEL and REFIND_BOOT_PREFIX idea in /etc/default/refind-kernel-hook.conf. It kind of adheres to the KISS rule.

What we would need besides setting these two variables that would default to `Void Linux` and `""` respectively would be some kind of a snippet that would, for instance, find the available and explicitly installed versions of linux in REFIND_BOOT_PREFIX and add these entries to the refind.conf file and preferably remove them also when they are uninstalled (so far it has been the job of the hooks). From what I understand, we still need some kind of a trigger. If I've misinterpreted your idea, please correct me.

Left-over kernels (the ones that vkpurge can deal with) can still be detected through the internal mode in refind.conf and the setting also_scan_dirs:
```
scanfor internal,manual
also_scan_dirs boot,void/ROOT/boot
```
this internal mode works in combination with /boot/refind_linux.conf which is independent of refind.conf:
```
"Boot with standard options"  "root=UUID=a9ee6e45-d348-4214-9560-0d0c73ab8077 ro rootflags=subvol=void/ROOT mem_sleep_default=deep i915.modeset=1 video=1920x1080 loglevel=4"
"Boot to single-user mode"    "root=UUID=a9ee6e45-d348-4214-9560-0d0c73ab8077 ro rootflags=subvol=void/ROOT mem_sleep_default=deep i915.modeset=1 video=1920x1080 loglevel=4 single"
"Boot with minimal options"   "root=UUID=a9ee6e45-d348-4214-9560-0d0c73ab8077 ro rootflags=subvol=void/ROOT"
```
Such "subentries" populate only one "slot" when rEFInd is launched and can be accessed after pressing TAB whereas manual stanzas (the ones in refind.conf) are separate entities in the rEFInd menu. I realize manual vs internal may sound confusing but that's the way I've seen things work in rEFInd.

Reply:
Sorry, I think I have misread your post. Now I realize you referred to the formatting criteria. Sure, I'll drop the cosmetic changes.